crates/loam-cli/src/commands/dev/mod.rs

```diff
@@ -0,0 +1,208 @@
+use clap::Parser;
+use notify::{self, RecursiveMode, Watcher};
+use std::{
+    env, fs,
+    path::{Path, PathBuf},
+    sync::Arc,
+};
+use tokio::sync::mpsc;
+use tokio::sync::Mutex;
+use tokio::time;
+
+use crate::commands::build;
+
+use super::build::clients::LoamEnv;
+
+struct RebuildState {
+    pending: bool,
```

Comment:
Similar question as chad here. Is this needed or can it just be a bool?

Reply:
changed to just a bool